test/integration/manifests/load/privileged-daemonset.yaml

```diff
@@ -24,15 +24,22 @@ spec:
             privileged: true
             runAsUser: 0
           volumeMounts:
-            - mountPath: /var/run/azure-cns
+            - mountPath: /var/run/azure-network

```

Comment:
This might break the existing cilium overlay load test pipeline fi privileged pods are used to get the json file. ( I don't think they do, but good to check)